src/main/java/com/monumental/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -232,6 +232,20 @@
                                 result.getWarnings().add("Cannot upload images with a .csv file. You must package your .csv and your images into a .zip file and upload it.");
                             }
                             break;
+                        case "imageReferenceUrls":
+                            if (zipFile != null) {
+                                result.getImageReferenceUrls().addAll(parseCsvArray(value, true));
+                            } else {
+                                result.getWarnings().add("Cannot upload images with a .csv file. You must package your .csv and your images into a .zip file and upload it.");
```

Comment:
Won't this just create duplicate warnings? not sure I see the point of checking for a zip file here.

Reply:
going to modify the message for clarity.  I think we should still include a message though, in case the user tries to send image reference URLs without any images.  In that case, this is the only warning they would get